src/sentry/event_manager.py

```diff
@@ -691,8 +691,9 @@ def save(self, project, raw=False):
             ), timestamp=event.datetime)
 
         if is_new and release:
-            buffer.incr(Release, {'new_groups': 1}, {
-                'id': release.id,
+            buffer.incr(ReleaseProject, {'new_groups': 1}, {
```

Comment:
Basiaclly, we need buffers to translate to something like:

```sql
UPDATE sentry_release_project SET test = COALESCE(test, 0) + 1;
```

Unfortunately, Django does not have a `COALESCE` in the version we use. :( Not sure what this will look like for MySQL or sqlite.